segment_language.go

```diff
@@ -18,6 +23,21 @@ func (l *language) string() string {
 }
 
 func (l *language) enabled() bool {
+
+	displayMode := l.props.getString(DisplayMode, "when_in_context")
+	displayVersion := l.props.getBool(DisplayVersion, true)
+	hasVersion := l.getVersion()
+
+	if displayMode == "always" {
```

Comment:
These would need to be a specific type too: `type DisplayMode string`. I would also opt to change `when_in_context` to `context`.